temp/prometheus-builder/build.sh

```diff
@@ -2,30 +2,38 @@
 
 PR_NUMBER=$1
 FLAGS="${@:2}"
-
-if [ -z "$PR_NUMBER" ]; then echo "ERROR::PR NUMBER is missing in argument" && exit 1; fi
-
 DIR="/go/src/github.com/prometheus/prometheus"
+VOLUME_DIR="/prometheus-builder" # from 3_prometheus-test.yaml
```

Comment:
can't we pass this as an env variable so it is all in `3_prometheus-test.yaml` this we we need to remember to change this only in a single file.

Reply:
yes! we can do this. I'll add it. this will also avoid rebuilding the image if we change the `mountPath`